### PR TITLE
Non tag reported as error

### LINKED ISSUE #443: Non tag reported as error
Non tag reported as error

Given this template:

`{{data1}}`

The fault tolerant parser consider that `{data1}` is an expression (and ls report a warning error). The parser should generate a text node in this case instead of expression node.

See `Problem7` from https://github.com/redhat-developer/quarkus-ls/issues/432

### PR DESCRIPTION
**Pull request by angelozerr**
Non tag reported as error

Fixes #443

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

File: qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/parser/template/scanner/TemplateScanner.java [hunk 1]
```diff
@@ -255,6 +257,10 @@ protected TokenType internalScan() {
 		finishToken(offset, TokenType.Unknown, errorMessage);
 	}
 
+	private boolean isValidIdentifierStart(int ch) {
+		return Character.isDigit(ch) || Character.isAlphabetic(ch) || ch == '_' || ch == '}';
```
Comment (fbricon) [inline qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/parser/template/scanner/TemplateScanner.java:261]: looks weird to say } is a valid identifier start character

Comment (angelozerr) [reply qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/parser/template/scanner/TemplateScanner.java:261]: Qute code use this name https://github.com/quarkusio/quarkus/blob/4940f063022e3fc8fc9dd723d1b58f4ca4b2c4d3/independent-projects/qute/core/src/main/java/io/quarkus/qute/Parser.java#L309

Comment (fbricon) [reply qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/parser/template/scanner/TemplateScanner.java:261]: yes, and } is not in there

```java
// A valid identifier must start with a digit, alphabet, underscore, comment delimiter, cdata start delimiter or a tag command (e.g. # for sections)
        return Tag.isCommand(character) || character == COMMENT_DELIMITER || character == CDATA_START_DELIMITER
                || character == CDATA_START_DELIMITER_OLD
                || character == UNDERSCORE
                || Character.isDigit(character)
                || Character.isAlphabetic(character);
```

Comment (angelozerr) [reply qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/parser/template/scanner/TemplateScanner.java]: Good catch! I fixed and write a test for that.

**Review by jeffmaury — APPROVED** [review]
LGTM

**Comment by fbricon** [comment]
Thanks @angelozerr

**Pull request merged**